grpc/teststate_test.go

```diff
@@ -114,6 +116,11 @@ func newTestState(t *testing.T) testState {
 	logger.SetLevel(logrus.InfoLevel)
 	logger.Out = io.Discard
 
+	hook := testutils.NewLogHook(
+		logrus.InfoLevel, logrus.ErrorLevel, logrus.FatalLevel, logrus.PanicLevel,
+	)
```

Comment:
If we use LogContains then we don't need it, we can just use the default version.
```suggestion
	hook := testutils.NewLogHook()
```

Reply:
@olegbespalov is it relevant to exclude the Warning level here?

Reply:
For this particular change, the Warning level is irrelevant, but it's not harmful to include all levels, so going to adjust it to just default constructor